dll/win32/shell32/shell32.spec

```
@@ -466,3 +466,4 @@
 754 stub -noname SHLimitInputEditWithFlags
 755 stdcall -noname PathIsEqualOrSubFolder(wstr wstr)
 756 stub -noname DeleteFileThumbnail
+757 stdcall -version=0x600+ DisplayNameOfW(ptr ptr long wstr long)
```

```suggestion
757 stdcall -version=0x600+ DisplayNameOfW(ptr ptr long ptr long)
```
because the string is an output buffer. If it were input, it would have been `wstr`.